src/fpm_targets.f90

```diff
@@ -206,6 +229,8 @@ subroutine build_target_list(targets,model)
                     end if
 
                 case (FPM_UNIT_PROGRAM)
+                    sources(i)%flags=model%cmd_compile_flags//" "//get_file_scope_flags(sources(i), profile)//profile%flags
```

Comment:
It looks like `FPM_UNIT_PROGRAM` has slightly different behaviour to the other objects with respect to file-scope flags. For other objects, you EITHER have file-scope flags OR profile flags, but for `FPM_UNIT_PROGRAM` you can have file-scope flags AND profile flags. Is this intentional?

Reply:
No, I just forgot to change this, thanks.